app/views/pets/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <% @pets.each do |pet| %>
   <img src="<%= pet.image_path %>" alt="<%= pet.image_path %>">
   <ul>
-    <li>Name: <%= pet.name %></li>
+    <li>Name: <%= link_to "#{pet.name}", "/pets/#{pet.id}"%></li>
```

Comment:
links to each individual pet show page.
I would like to style these links so they look more inviting,.